### PR TITLE
add circuitpython compilation section to README

### PR DESCRIPTION
**Pull request by v923z**

### CODE REVIEW

**Comment by v923z** [comment]
@dhalbert Is this OK, or is something still missing?

File: README.md [hunk 1]
```diff
@@ -13,9 +13,10 @@ The `float` implementation of `micropython` (`float`, or `double`) is automatica
 1. [Benchmarks](#benchmarks)
 1. [Firmware](#firmware)
     1. [UNIX](#unix-port)
-    2. [STM-based boards](#stm-based-boards)
-    3. [ESP32-based boards](#esp32-based-boards)
-    4. [RP2-based boards](#rp2-based-boards)
+    1. [STM-based boards](#stm-based-boards)
+    1. [ESP32-based boards](#esp32-based-boards)
+    1. [RP2-based boards](#rp2-based-boards)
+    1. [Compiling for circuitpython](#compiling-for-circuitpython)
```
Comment (dhalbert) [inline README.md:19]:
```suggestion
    1. [Compiling for CircuitPython](#compiling-for-circuitpython)
```

File: README.md [hunk 2]
```diff
@@ -316,6 +317,29 @@ make USER_C_MODULE=/path/to/ulab/code/micropython.cmake
 
 If `micropython` and `ulab` were in the same folder on the computer, you can set `USER_C_MODULES=../../../ulab/code/micropython.cmake`. The compiled firmware will be placed in `micropython/ports/rp2/build`.
 
+# Compiling for circuitpython
```
Comment (dhalbert) [inline README.md:320]:
```suggestion
# Compiling for CircuitPython
```

**Review by dhalbert — CHANGES_REQUESTED** [review]
Thanks for adding this. Just a suggestion of some capitalization fixes.

**Comment by v923z** [comment]
@dhalbert Many thanks, Dan!

**Pull request merged**